src/server/api/controllers/organizations.controller.js

```diff
@@ -1,33 +1,19 @@
-const HttpError = require('../lib/utils/http-error');
 const knex = require('../../config/db');
 
 const getOrganizations = async () => {
   try {
-    return await knex('organizations').select('id', ' name', 'imageUrl');
-  } catch (err) {
-    console.log(err);
-  }
-};
+    const organizations = await knex('organizations').select(
+      'id',
+      'name',
+      'image_url',
```

Comment:
You need to return this back as `imageUrl`.

Reply:
corrected as `organizations.image_url as imageUrl `